Asterix/wfsc/wf_control_functions.py

```diff
@@ -600,8 +601,11 @@ def create_singlewl_interaction_matrix(testbed: Testbed,
                 plt.ioff()
             # We save the interaction matrix:
             if (initial_DM_voltage == 0.).all():
-                fits.writeto(os.path.join(matrix_dir, fileDirectMatrix + ".fits"),
-                             InterMat[:, pos_in_matrix:pos_in_matrix + DM.basis_size])
+                hdu_list = fits.HDUList([fits.PrimaryHDU(header=header)])
+                hdu_list.append(
+                    fits.ImageHDU(data=InterMat[:, pos_in_matrix:pos_in_matrix + DM.basis_size], name='MATRIX'))
```

Comment:
- Where does the transpose happen that is needed to feed this into `thd-controls`? Is this elsewhere? Could you point me to it please?

- We both agreed that "BOSTON" is not an ideal keyword value here, but to make it work with `thd-controls` for now, "MATRIX" needs to be replaced by "BOSTON". You could open an issue on `thd-controls` to discuss (and change it) later on, so that we don't forget about it.

Reply:
This is internal to asterix here. The place where we save for thd controls is at the same place as we save for labview, in "corrector.py"